ntbs-integration-tests/NotificationPages/HospitalDetailsPageTests.cs

```diff
@@ -294,6 +305,46 @@ public async Task PostNotifiedChangingCaseManager_RedirectToNextPage_IfModelIsVa
             Assert.Equal(HttpStatusCode.Redirect, result.StatusCode);
         }
 
+        [Fact]
+        public async Task OnGetNotifiedWithInactiveCaseManager_HasCaseManagerInDropdown()
```

Comment:
Do we have a test already checking the inactive case manager isn't in the dropdown normally?

Reply:
No I guess not, the tests that already exist just check the return from filtering doesn't include it, with a method in this file called AssertAreGatesheadCaseManagers. I'll add a test for the initial dropdown